src/components/BorderCountry.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from 'react';
+import { useTheme } from '../hooks/themeUtils';
+import style from './BorderCountry.module.scss';
+import { API_URL2 } from '../config';
```

Comment:
Can we give it a more readable name? 

Now it's unclear why we're using API_URL2 but not 1.